src/toolkits/activity_classification/class_registrations.cpp

```diff
@@ -17,6 +17,24 @@ namespace activity_classification {
 BEGIN_FUNCTION_REGISTRATION
 REGISTER_FUNCTION(_activity_classifier_prepare_data , "data" , "features" , "session_id" , "prediction_window" , "predictions_in_chunk" , "target")
 REGISTER_FUNCTION(_activity_classifier_prepare_data_verbose , "data" , "features" , "session_id" , "prediction_window" , "predictions_in_chunk" , "target")
+REGISTER_NAMED_FUNCTION("_activity_classifier_random_split_by_session", activity_classifier::random_split_by_session, "data","session_id_column_name", "fraction", "seed");
```

Comment:
Let's use "session_id" instead of "session_id_column_name", for consistency with the `train` method (C API naming) on `activity_classifier`.

Reply:
train method uses ```session_id_column_name```

Reply:
Got it